include/torch-mlir/Dialect/Torch/IR/TorchOps.h

```diff
@@ -267,6 +268,41 @@ m_TorchTensorSizeInt(Value tensor, int64_t *dim) {
   return detail::torch_tensor_size_int_op_binder(tensor, dim);
 }
 
+namespace detail {
+/// Matches the constant integers stored in a `onnx.Constant`.
+struct onnx_list_of_constant_ints_op_binder {
+  SmallVectorImpl<int64_t> &bind_values;
+
+  /// Creates a matcher instance that binds the value to bvs if match succeeds.
+  onnx_list_of_constant_ints_op_binder(SmallVectorImpl<int64_t> &bvs)
+      : bind_values(bvs) {}
+
+  bool match(Operation *op) {
+    auto constOp = dyn_cast<Torch::OperatorOp>(op);
+    if (!constOp || !constOp.getName().equals("onnx.Constant"))
+      return false;
+
```

Comment:
You should include the endianness check here so we don't assume things are fine on a wrong endian system